### PR TITLE
Homepage/ Video listing page

### PR DESCRIPTION
**Pull request by Pratik1005**
Created video listing page UI which have: 

- sidebar menu
- category chips
- video cards

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (rahulmistry751) [inline public/index.html]: change the description

**Review by rahulmistry751 — APPROVED** [review]
Good folder structure and error handling

File: src/components/NavMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          <li>
+            <span class="material-icons">history</span> History
+          </li>
+          <li>
+            <span class="material-icons">watch_later</span>
+            Watch later
+          </li>
+          <li>
+            <span class="material-icons">dark_mode</span>
+            Dark mode
```
Comment (manismk) [inline src/components/NavMenu.jsx]: Change class to className

**Review by manismk — APPROVED** [review]
Looks Good. Good folder structure

**Pull request merged**